cmd/lakefs/cmd/root.go

```diff
@@ -109,6 +109,9 @@ func initConfig() {
 	}
 
 	logger.WithFields(cfg.ToLoggerFields()).Info("Config")
+	if viper.GetBool("database.alpha_kv_enabled") {
```

Comment:
Prefer to use the configuration structure - it will hold refactor tools and 'find usage'